cime_config/machines/cmake_macros/gnu_theta.cmake

```diff
@@ -1,13 +1,4 @@
-if (NOT DEBUG)
-  string(APPEND CFLAGS " -O2")
-endif()
-if (DEBUG)
-  string(APPEND CFLAGS " -O0")
-endif()
-if (NOT DEBUG)
-  string(APPEND FFLAGS " -O2")
-endif()
-if (DEBUG)
-  string(APPEND FFLAGS " -O0")
-endif()
-string(APPEND CXX_LIBS " -lstdc++")
+string(APPEND CMAKE_C_FLAGS_RELEASE " -O2")
+string(APPEND CMAKE_C_FLAGS_DEBUG " -O0")
```

Comment:
I comment here, but could be on any file. Since gnu follows the rule "last opt flag wins", and since `-O2` seems to be the most common choice for RELEASE, perhaps we can just set it in `gnu.cmake`, and add a line to `gnu_XYZ.cmake` only if they need different flags?